### PR TITLE
Add contribution link & badge to README

### PR DESCRIPTION
**Pull request by kwalrath**

### CODE REVIEW

**Comment by chalin** [comment]
Cool!

File: README.md [hunk 1]
```diff
@@ -1,10 +1,13 @@
 # The webdev site
 
 [![Build Status](https://travis-ci.org/dart-lang/site-webdev.svg?branch=master)](https://travis-ci.org/dart-lang/site-webdev)
+[![first-timers-only](http://img.shields.io/badge/first--timers--only-friendly-blue.svg?style=flat-square)](http://www.firsttimersonly.com/)
 
 This repo implements the [webdev.dartlang.org](http://webdev.dartlang.org) website. It uses the tools and infrastructure of
 www.dartlang.org (which is implemented in [github.com/dart-lang/site-www](https://github.com/dart-lang/site-www)).
 
+[We welcome contributions](CONTRIBUTING.md), even if you've never committed to GitHub before.
```
Comment (chalin) [inline README.md]: Why single out "never committed to GitHub before"?  How about
> [We welcome contributions](CONTRIBUTING.md), and we're [first-timer friendly](http://www.firsttimersonly.com)!

Should we be putting in a request to be added to https://github.com/up-for-grabs/up-for-grabs.net?

Comment (kwalrath) [inline README.md]: Good points! I'll do those things.

**Comment by kwalrath** [comment]
I've changed the text to match Patrice's suggestion, and I also added a heading and short intro for the build section.

**Comment by kwalrath** [comment]
Filed #832 about adding this repo to up-for-grabs.

**Pull request merged**